lineman/app/components/thread_preview/thread_preview.scss

```diff
@@ -8,72 +8,121 @@
   }
 }
 
+.thread-preview__avatar {
+  display: inline-block;
+  width: 10%;
+  .placeholder {
+    border-radius: 4px;
```

Comment:
This I can do, although it is really just a temp class for demonstration purposes.